src/apps/backend/modules/task/internal/store/task-db.ts

```diff
@@ -6,6 +6,8 @@ export interface TaskDB {
   active: boolean;
   description: string;
   title: string;
+  sharedWith: Types.ObjectId;
+  comments: Types.ObjectId;
```

Comment:
Since a task can have non trivial number of comments, it is not wise to store this as an array in the collection, rather a new collection should be created to store association between task and comment